### PR TITLE
Jordan/kava theme

### PR DESCRIPTION
**Pull request by jbibla**
**Description:**

- implemented proper kava theme and some related style improvements noted by ana
Screen Shot 2020-05-05 at 9 55 24 PM

Thank you! 🚀

---

For contributor:

- [ ] Added changes entries. Run `yarn changelog` for a guided process.
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

**Comment by Bitcoinera** [comment]
Interesting. I thought this one was going to come out with red as the main color?

Also kind of trippy the navbar on the left, with psychedelic red and purple (not suitable for epileptic people)

image

**Comment by Bitcoinera** [comment]
But we can merge it and add some entropies to it xD

**Comment by jbibla** [comment]
the kava red is not dark enough so the text has to be black for it to pass contrast accessiblity tests — which isn't super nice.

**Comment by Bitcoinera** [comment]
> yeah, maybe too psychedelic? i was tired 😄

xD

We can just make their read a bit darker? Or maybe this theme but less psychedelic 😅 

We could also ask in our Telegram group see how people feel 😆 Maybe they like psychedelic

**Comment by jbibla** [comment]
yeah it's ok for now. includes some themes fixes and kava isn't even live yet. i'd say we just merge and get some feedback when we announce the integration.

File: src/components/common/AppHeader.vue [hunk 1]
```diff
@@ -3,11 +3,35 @@
...
+            <path
+              d="M231.218 56.489C231.218 48.888 234.476 44.2053 241.534 44.2053C248.321 44.2053 250.56 48.413 250.56 55.0638V76.238H260.061V52.7564C260.061 42.6443 254.632 36.1971 244.656 36.1971C237.598 36.1971 232.847 39.7261 230.608 45.1554L229.725 37.2151H221.717V76.238H231.218V56.489Z"
+            />
+            <path
+              d="M271.775 32.668C275.914 32.668 278.018 30.9035 278.018 26.5601C278.018 22.0809 275.847 20.52 271.775 20.52C267.771 20.52 265.599 22.0809 265.599 26.5601C265.599 31.1071 267.295 32.668 271.775 32.668ZM276.457 76.238V37.2151H266.888V76.238H276.457Z"
+            />
+            <path
+              d="M303.231 43.5945C309.882 43.5945 313.411 46.9878 314.158 52.96H291.355C292.372 46.9878 296.512 43.5945 303.231 43.5945ZM322.98 64.7008H312.8C311.307 67.8905 308.321 69.655 303.163 69.655C296.037 69.655 292.101 66.3296 291.287 60.2216H323.523V56.0818C323.523 45.4947 317.076 36.1971 303.095 36.1971C289.929 36.1971 281.989 44.5446 281.989 56.8283V57.5749C281.989 69.5872 288.504 77.3239 302.688 77.3239C313.411 77.3239 320.265 72.9126 322.98 64.7008Z"
+            />
+          </svg>
```
Comment (Bitcoinera) [inline src/components/common/AppHeader.vue:34]: I'm just curious, why do we switch to SVG for our logo? To not need to import it right?

File: src/components/network/PoweredBy.vue [hunk 2]
```diff
@@ -27,17 +28,15 @@
         inactive: !network.powered.providerAddress
       }"
       @click="
-        network.powered.providerAddress && isCurrentNetwork
+        isMenu
```
Comment (Bitcoinera) [inline src/components/network/PoweredBy.vue:31]: ?

Comment (Bitcoinera) [reply src/components/network/PoweredBy.vue:31]: ah ok, yeah. Maybe it is just better like that. Not having that link in Networks 👍 Yes, good change

File: src/components/network/PoweredBy.vue [hunk 3]
```diff
@@ -4,7 +4,8 @@
     class="powered-by-container"
     :class="{
       'powered-by-line': poweredByLine,
-      'hide-on-mobile': hideOnMobile
+      'hide-on-mobile': hideOnMobile,
+      'on-menu': isMenu
```
Comment (Bitcoinera) [inline src/components/network/PoweredBy.vue:8]: I hope validators don't get annoyed that we are not displaying this on mobile 😅 

Should be OK though right

File: src/components/common/AppHeader.vue (hunk 1 above)
Comment (jbibla) [reply src/components/common/AppHeader.vue:34]: good question! it's to be able to set the color with a variable.

File: src/components/network/PoweredBy.vue (hunk 2 above)
Comment (jbibla) [reply src/components/network/PoweredBy.vue:31]: yeah so now it's only a link when it's in the sidebar.

File: src/components/network/PoweredBy.vue (hunk 3 above)
Comment (jbibla) [reply src/components/network/PoweredBy.vue:8]: oh yeah - i didn't even realize that.

Comment (Bitcoinera) [reply src/components/network/PoweredBy.vue:8]: xD

Comment (Bitcoinera) [reply src/components/network/PoweredBy.vue:8]: Well we can revert 😅  Maybe I was too quick

**Pull request merged**